doc/terminology.rst

```diff
@@ -0,0 +1,51 @@
+.. _terminology:
+
+.. https://github.com/pydata/xarray/issues/2410
+.. https://github.com/pydata/xarray/issues/1295
```

Comment:
Forgive me for my lack of docs understanding—what do these do?

Reply:
These comments were just for myself: issues where Xarray contributors or maintainers defined words. Removing them now.